packages/webview_flutter/lib/webview_flutter.dart

```diff
@@ -469,6 +469,22 @@ class WebViewController {
   }
 }
 
+/// Manages cookies pertaining to all [WebView]s.
+class CookieManager {
+  const CookieManager._();
+
+  static const MethodChannel _channel =
+      MethodChannel('plugins.flutter.io/cookie_manager');
+
+  /// Clears all cookies.
+  ///
+  /// This is supported for >= IOS 9 and Android api level >= 16.
+  /// returns true if cookies were present before clearing, else false.
+  Future<bool> clearCookies() async {
+    return await _channel.invokeMethod<bool>('clearCookies');
```

Comment:
Using the type parameter will make this incompatible with stable, see the other plugin repos

Reply:
good catch, these are risky days for missing these on code reviews. 😨 
https://github.com/flutter/flutter/issues/27110 would have been helpful here.

Reply:
Ouch, fixed this. Thanks for the flag!